content/docs/user-guide/managing-external-data.md

```diff
@@ -1,4 +1,4 @@
-# External Outputs
+# Managing External Data
```

Comment:
This change implies there's going to be a distinction between "external inputs" and "external outputs." Not sure if that's the intention.

Reply:
This is a roll-back actually. The page title was changed accidentally at some point.

Reply:
Context: https://github.com/iterative/dvc.org/pull/2154#pullrequestreview-610353476

Reply:
BTW notice how the file name was already managing-external-data.md (resolving this one).